RocketControlUnitGUI/src/data.ts

```diff
@@ -0,0 +1,110 @@
+  // data.ts
+  import PocketBase from 'pocketbase';
+  import dotenv from "dotenv";
+
+  
+
+  const ADMIN_EMAIL: string = import.meta.env.VITE_EMAIL;
+  const ADMIN_PASSWORD: string = import.meta.env.VITE_PASSWORD;
+
+ console.log(ADMIN_EMAIL);
+
+  const pb = new PocketBase('http://127.0.0.1:8090');
```

Comment:
Duplicate pocket base connection, the usePocketbase hook in master solves this issue as long as we pass around the same instance.